patches.json

```diff
@@ -1677,5 +1677,17 @@
         }
       }
     }
+  },
+  "MC-41359": {
+    "magento/magento2-base": {
+      "Fixes the issue where SameSite cookie param was set incorrectly.": {
+        ">=2.3.6 <2.3.7": {
```

Comment:
@ihor-sviziev: Pretty sure this needs to be `2.3.6-p1` instead of `2.3.6`
The 2.4.x versions seem to be correct!

Reply:
Fixed